docker/linea-sepolia/maru/config/maru-genesis.json

```diff
@@ -0,0 +1,26 @@
+{
+    "chainId": 59141,
+    "config": {
+      "0": {
+        "type": "difficultyAwareQbft",
+        "blockTimeSeconds": 2,
+        "postTtdConfig": {
+          "validatorSet": ["0xb4323e9d9998990e6333293f8f0d2d09d1619328"],
```

Comment:
In the [migration guide](https://docs.linea.build/get-started/how-to/run-a-node/beta-v4-migration) the validator set is different, which one is correct?

Reply:
`0xb4323e9d9998990e6333293f8f0d2d09d1619328` is the old one